Integrations/git/git.py

```diff
@@ -0,0 +1,306 @@
+
+
+''' REQUESTS FUNCTIONS '''
+
+
+def create_issue(title, body, milestone, labels, assignees):
+    if title == "":
+        return_error("Error: No title given for created issue")
+
+    data = data_formatting(title=title, body=body, milestone=milestone, labels=labels, assignees=assignees, state=None)
+    response = http_request(method='POST', url_suffix=USER_SUFFIX + ISSUE_SUFFIX, data=data)
+    return response
+
+
+def close_issue(issue_number):
+    response = http_request(method='PATCH', url_suffix=USER_SUFFIX + ISSUE_SUFFIX + '/{}'.format(str(issue_number)),
+                            data={'state': 'closed'})
+    return response
+
+
+def update_issue(issue_number, title, body, state, milestone, labels, assign):
+    data = data_formatting(title=title, body=body, milestone=milestone, labels=labels, assignees=assign, state=state)
+    response = http_request(method='PATCH', url_suffix=USER_SUFFIX + ISSUE_SUFFIX + '/{}'.format(str(issue_number)),
+                            data=data)
+    if response.get('errors'):
+        return_error(response.get('errors'))
+
+    return response
+
+
+def list_all_issue(only_open):
+    params = {}  # type: dict
+    if only_open is False:
+        params = {'state': 'all'}
+
+    response = http_request(method='GET', url_suffix=USER_SUFFIX + ISSUE_SUFFIX, params=params)
+    return response
+
+
+def search_issue(query):
+    response = http_request(method='GET', url_suffix='/search/issues', params={'q': query})
+    if response.get('errors'):
+        return_error(response.get('errors'))
+
+    return response
+
+
+def get_download_count():
+    response = http_request(method='GET', url_suffix=USER_SUFFIX + RELEASE_SUFFIX)
+    count_per_release = []
+    for release in response:
+        total_download_count = 0
+        for asset in release['assets']:
+            total_download_count = total_download_count + asset['download_count']
+
+        release_info = {
+            'URL': release.get('url'),
+            'Download_count': total_download_count
+        }
+        count_per_release.append(release_info)
+
+    context_create_release(release_list=count_per_release, response=response)
+
+
+''' COMMANDS MANAGER / SWITCH PANEL '''
+
+
+def create_command():
+    args = demisto.args()
+    response = create_issue(args.get('title'), args.get('body'),
+                            args.get('milestone'), args.get('labels'), args.get('assignees'))
+    issue_format(response, result_token=True)
+
+
+def close_command():
+    issue_number = demisto.args().get('issue_number')
+    response = close_issue(issue_number)
+    issue_format(response, result_token=True)
+
+
+def update_command():
+    args = demisto.args()
+    response = update_issue(args['issue_number'], args.get('title'), args.get('body'), args.get('state'),
+                            args.get('milestone'), args.get('labels'), args.get('assignees'))
+    issue_format(response, result_token=True)
```

Comment:
I'm not sure that i get the usage of result_token

Reply:
I did explain it under context_create_issue.
if it set to true, the issue_format function gets the results to Demisto.
if false it just returns the dictionary representing a single issue - to be appended to issue table (like in list-all-issues or search-issue)